lib/src/model/package_builder.dart

```diff
@@ -386,52 +388,57 @@ class PubPackageBuilder implements PackageBuilder {
   List<String> _includedExternalsFrom(Iterable<String> files) => [
         for (var file in files)
           ...DartdocOptionContext.fromContext(
-            config,
-            config.resourceProvider.getFile(file),
-            config.resourceProvider,
+            _config,
+            _config.resourceProvider.getFile(file),
+            _config.resourceProvider,
           ).includeExternal,
       ];
 
-  Future<Set<String>> _getFiles() async {
-    var files = config.topLevelPackageMeta.isSdk
+  /// Returns the set of files that may contain elements that need to be
+  /// documented.
+  ///
+  /// This takes into account the 'auto-include-dependencies' option, the
+  /// 'exclude' option, and the 'include-external' option.
+  Future<Set<String>> _getFilesToDocument() async {
+    var files = _config.topLevelPackageMeta.isSdk
         ? _sdkFilesToDocument
         : await _findFilesToDocumentInPackage(
-            config.inputDir,
-            includeDependencies: config.autoIncludeDependencies,
+            _config.inputDir,
+            includeDependencies: _config.autoIncludeDependencies,
             filterExcludes: true,
           ).toList();
     files = [...files, ..._includedExternalsFrom(files)];
     return {
-      ...files.map((s) => resourceProvider.pathContext
-          .absolute(resourceProvider.getFile(s).path)),
-      ...getEmbedderSdkFiles(),
+      ...files
+          .map((s) => _pathContext.absolute(_resourceProvider.getFile(s).path)),
+      ..._embedderSdkFiles,
     };
   }
 
-  Iterable<String> getEmbedderSdkFiles() {
-    return [
-      for (var dartUri in _embedderSdkUris)
-        resourceProvider.pathContext.absolute(resourceProvider
-            .getFile(embedderSdk!.mapDartUri(dartUri)!.fullName)
-            .path),
-    ];
-  }
+  Iterable<String> get _embedderSdkFiles => [
+        for (var dartUri in _embedderSdkUris)
+          _pathContext.absolute(_resourceProvider
+              .getFile(_embedderSdk!.mapDartUri(dartUri)!.fullName)
+              .path),
+      ];
 
-  bool get hasEmbedderSdkFiles => _embedderSdkUris.isNotEmpty;
+  bool get _hasEmbedderSdkFiles => _embedderSdkUris.isNotEmpty;
```

Comment:
I feel like this doesn't even have to be it's own getter and can just be inlined in the one place it's used.

Reply:
Ha, even better. Done.